servicedirectory/spec/quickstart_spec.rb

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/cloud/service_directory/v1beta1"
+
+describe "Service Directory Registration Service Quickstart" do
+  ServiceDirectory = Google::Cloud::ServiceDirectory::V1beta1
```

Comment:
Is it possible to use the wrapper gem and factory methods here? (i.e. 
```require "google/cloud/service_directory"

...
registration_service = Google::Cloud::ServiceDirectory.registration_service
```

Reply:
Done. I originally missed this file when updating all the other ones with the wrapper gem, sorry!